src/models/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'production';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
```

Comment:
Consider separating your imports/exports from the rest of the code with new lines.

Reply:
resolved